server/router/fileserver/README.md

```
@@ -1,304 +1,46 @@
-
-```
-fileserver/
-├── fileserver.go           # Main service and HTTP handlers
-├── README.md              # This file
-└── fileserver_test.go     # Tests (to be added)
-```
-
-## API Endpoints
-
-### 1. Attachment Binary
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)